scripts/cicd/verify-cicd-operation.py

```diff
@@ -7,7 +7,7 @@
 # pylint: disable=bare-except
 
 #Jenkins:
-#ssh use-tower1.ops.rhcloud.com <operation>
+#ssh use-tower1.ops.rhcloud.com <operation> [approved_arg1] [approved_arg2] ...
```

Comment:
Comment would do well to address - what are approved args?  what are they used for?  how are they approved?